src/hevm/src/EVM/SymExec.hs

```diff
@@ -261,7 +272,10 @@ type Precondition = VM -> Prop
 type Postcondition = VM -> Expr End -> Prop
 
 checkAssert :: SolverGroup -> [Word256] -> ByteString -> Maybe (Text, [AbiType]) -> [String] -> IO [VerifyResult]
-checkAssert solvers errs c signature' concreteArgs = verifyContract solvers c signature' concreteArgs SymbolicS Nothing (Just $ checkAssertions errs)
+checkAssert a b c d e  = checkAssert' a b c d e defaultVeriOpts
```

Comment:
can we  keep the meaningful arg names here?

Reply:
Do we even need this variant at all? Can we just construct a `VeriOpts` and pass that in whereever we call `checkAssert`?

Reply:
Good point, actually. Fixed now :)